lib/Dialect/AIEVec/Transforms/ConvertVectorToAIEVec.cpp

```diff
@@ -38,6 +38,7 @@ namespace xilinx::aievec {
 } // namespace xilinx::aievec
 
 using namespace mlir;
+using namespace mlir::affine;
```

Comment:
I'd rather leave the `affine::` prefix everywhere, for clarity. I suspect the next step is to change the op names from `Affine<op>` to simply `<op>`, and having things like `ForOp` floating around obscures the type.